app/javascript/controllers/index.js

```diff
@@ -6,3 +6,6 @@ import { application } from "./application"
 
 import HelloController from "./hello_controller"
 application.register("hello", HelloController)
```

Comment:
What is this HelloController for? We should have meaningful names for controllers.